.github/workflows/copilot-deploy-memorypipeline.yml

```diff
@@ -0,0 +1,76 @@
+﻿name: copilot-deploy-memorypipeline
```

Comment:
copilot-deploy-memoryservice would probably be a better name

Reply:
I will stick with pipeline for now, since memory service may mean an entire service in semantic memory that has the ask endpoint, whereas pipeline is just the processing pipeline.